lib/streamlit/elements/arrow_altair.py

```diff
@@ -757,8 +765,9 @@ def altair_chart(
             The Altair chart object to display.
 
         use_container_width : bool
-            If True, set the chart width to the column width. This takes
-            precedence over Altair's native ``width`` value.
+            If True, set the chart width to the width of the parent container.
+            This takes precedence over Altair's native ``width`` value.
```

Comment:
super nit: Should this be `spec's native width`? That seems more correct than `Altair's native width` because idk what Altair's native width is. It's usually defined in the spec.

Reply:
good point, changed it 👍